docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md

```diff
@@ -1,48 +1,158 @@
-import OptionalProfileYourDataToGenerateExpectationsThenEditThemInInteractiveMode from './components_how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/_optional_profile_your_data_to_generate_expectations_then_edit_them_in_interactive_mode.mdx'
-import SaveABatchRequestToReuseWhenEditingAnExpectationSuiteInInteractiveMode from './components_how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/_save_a_batch_request_to_reuse_when_editing_an_expectation_suite_in_interactive_mode.mdx'
-import UseTheBuiltInHelpToReviewTheCliSSuiteNewOptionalFlags from './components_how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/_use_the_builtin_help_to_review_the_clis_suite_new_optional_flags.mdx'
+import Prerequisites from '/docs/components/_prerequisites.jsx'
+import PrerequisiteQuickstartGuideComplete from '/docs/components/prerequisites/_quickstart_completed.mdx'
+import IfYouStillNeedToSetupGX from '/docs/components/prerequisites/_if_you_still_need_to_setup_gx.md'
+import DataContextInitializeQuickOrFilesystem from '/docs/components/setup/link_lists/_data_context_initialize_quick_or_filesystem.mdx'
+import ConnectingToDataFluently from '/docs/components/connect_to_data/link_lists/_connecting_to_data_fluently.md'
 
-<Preface />
+In order to Validate data, we must first define a set of Expectations for that data to be validated against.  In this guide, we will walk you through processes of interactively creating and editing Expectations while simultaneously validating them against a Batch of data. By validating our Expectations as we define them we can quickly see if the Expectations we are setting are suitable for our data, and adjust them if necessary.
+
+:::tip Does this process edit my data?
+Not at all.  We call this process the "Interactive method" of creating and editing Expectations because validating the Expectations as we work requires interacting with a Batch of data.  This data is introspected to see if it fits the defined Expectation, but it is not edited: only examined.
+:::
+
+## Prerequisites
+
+<Prerequisites>
+
+- Great Expectations installed in a Python environment
+- Initialized a Filesystem Data Context to save your Expectations in
+- Created a Datasource from which to request a Batch of data for introspection
+- A passion for data quality
+
+</Prerequisites> 
+
+<details>
+<summary>
+
+### If you still need to set up Great Expectations...
+
+</summary>
+
+<IfYouStillNeedToSetupGX />
+
+</details>
+
+<details>
+<summary>
+
+### If you still need to initialize your Data Context...
+
+</summary>
+
+Please see the appropriate guide from the following:
+
+<DataContextInitializeQuickOrFilesystem />
+
+</details>
+
+<details>
+<summary>
+
+### If you still need to create a Datasource...
+
+</summary>
+
+Please reference the appropriate guide from the following:
```

Comment:
```suggestion
Select one of the following guides:
```
Is there an inappropriate guide? :)

![image](https://user-images.githubusercontent.com/104205257/231878510-57d9f538-739c-4867-97e7-2d1ef6b554b2.png)

Reply:
The inappropriate guides would be the guides that correspond to environments or source data systems that don't match the one they're using. :P